src/main/java/com/github/manolo8/darkbot/core/objects/facades/AssemblyMediator.java

```diff
@@ -101,6 +104,7 @@ public void update() {
         }
     }
 
+    @Getter
```

Comment:
```suggestion
```
This is not part of the api, it should be hidden away and never available or used.

Please stop relying on internals from plugins, separate api and implementation